### PR TITLE
Added show function for GR and Magick backends

### PR DESCRIPTION
**Pull request by alishdipani**
This PR contains the addition of the show function for GR and Magick backends. Further, this will contain work done during the second phase of GSoC 2019.

### CODE REVIEW

File: lib/rubyplot/artist/figure.rb [hunk 1]
```diff
@@ -109,7 +109,7 @@ def show
         Rubyplot.backend.canvas_height = @height
         Rubyplot.backend.canvas_width = @width
         Rubyplot.backend.figure = self
-        Rubyplot.backend.init_output_device(nil, device: nil)
```
Comment (v0dro) [inline lib/rubyplot/artist/figure.rb]: This is leading to copy pasting of code in the `write` and `show` functions. Abstract these into another method.

File: lib/rubyplot/backend/magick_wrapper.rb [hunk 2]
```diff
@@ -27,7 +33,8 @@ class MagickWrapper < Base
       PIXEL_MULTIPLIERS = {
         inch: 96,
         cm: 39.7953,
-        pixel: 1
+        pixel: 1,
+        point: 4/3
```
Comment (v0dro) [inline lib/rubyplot/backend/magick_wrapper.rb]: Please add a comment here about what  exactly is point.

Comment (alishdipani) [reply lib/rubyplot/backend/magick_wrapper.rb]: Done

File: lib/rubyplot.rb [hunk 3]
```diff
@@ -144,12 +144,23 @@ module Rubyplot
     :double_line_single_ended,
     :double_line_double_ended
   ].freeze
-  
-  class << self    
+
+  class << self
+
+    attr_accessor :iruby_inline
```
Comment (v0dro) [inline lib/rubyplot.rb]: Add docs saying what exactly this does and how it can be used. BTW you were going to push some method-specific docs right?

**Comment by v0dro** [comment]
I will merge this since the only change I suggested is a doc fix. Please a full PR with method-wise documentation soon.

File: lib/rubyplot.rb (hunk 3 above)
Comment (alishdipani) [reply lib/rubyplot.rb]: I will work on writing the docs soon and will make a PR for that too.

**Pull request merged**